project-base/src/Controller/Front/CartController.php

```diff
@@ -138,13 +149,35 @@ public function indexAction(Request $request)
         ]);
     }
 
-    public function boxAction()
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     */
```

Comment:
Can you please unify the presence/absence of return type? Here it's missing and in `deleteAjaxAction` it is.